scipy/special/meson.build

```diff
@@ -519,12 +524,5 @@ py3.install_sources(
   subdir: 'scipy/special'
 )
 
-# FIXME: fails with "Argument must be string or file."
-#py3.install_sources(
```

Comment:
This looks good to review now. The FIXME note also seems to be resolved in this diff.